obonet/read.py

```diff
@@ -32,7 +35,7 @@ def read_obo(path_or_file):
     edge_tuples = list()
 
     for term in terms:
-        is_obsolete = term.get('is_obsolete', 'false') == 'true'
+        is_obsolete = ignore_obsolete and term.get('is_obsolete', 'false') == 'true'
         if is_obsolete:
```

Comment:
perhaps keep `is_obsolete = term.get('is_obsolete', 'false') == 'true'` but change the next line to:

```python
if ignore_obsolete and is_obsolete:
```